src/worker/server/lambdaServer.go

```diff
@@ -60,8 +60,11 @@ func (s *LambdaServer) RunLambda(w http.ResponseWriter, r *http.Request) {
 			img := urlParts[1]
 			s.lambdaMgr.Get(img).Invoke(w, r)
 		} else {
-			w.WriteHeader(http.StatusInternalServerError)
-			w.Write([]byte("expected invocation format: /run/<lambda-name>"))
+			img := urlParts[1]
+			s.lambdaMgr.Get(img).Invoke(w, r)
```

Comment:
If the two branches of the if/else are the same, can we simplify down to one code path?

Hopefully we can retain the deleted error message, but do it in server.py when it's not WSGI and we don't have the expected number of parts in the URL.